tuxemon/animation.py

```diff
@@ -1,13 +1,18 @@
+from __future__ import annotations
 import logging
 from collections import defaultdict
 from math import sqrt, cos, sin, pi
 
 import pygame
+from typing import Any, Optional, Callable, Sequence, DefaultDict, List, Tuple,\
+    Mapping, Union
 
 __all__ = ("Task", "Animation", "remove_animations_of")
 
 from tuxemon.compat import Rect
 
+ScheduledFunction = Callable[[], Any]
```

Comment:
Why is `Any` here?  Im thinking a non-callable object would cause an exception when the callback are executed.

Reply:
It is a callable that can return anything. I could enforce that the return type is ``None`` if you prefer.

Reply:
i might just be unfamiliar with the syntax.  if this is just how to type a generic callable and its return values, then thats fine with me.